docs/source/nodes_and_pipelines/pipeline_introduction.md

```diff
@@ -34,12 +38,112 @@ variance_pipeline = pipeline(
     ]
 )
 ```
-</details>
 
-You can use `describe` to discover what nodes are part of the pipeline:
 
-<details>
-<summary><b>Click to expand</b></summary>
+## Create a new blank pipeline with the `kedro pipeline create` command
+
+When managing your Kedro project, we recommend organising your pipelines separately to achieve modularity. This approach allows for easy copying and reuse of pipelines within and between projects. Simply put: one pipeline, one folder. To help with that, you can use the following command (the pipeline name must adhere to [Python convention](https://realpython.com/python-pep8/#naming-conventions)):
```

Comment:
This also reads a little bit unclear to me. I think this part could explain the idea behind dividing tasks into pipelines. For eg, a project usually contains a lot of tasks and we recommend grouping tasks that frequently are executed together into individual pipelines. These pipelines should ideally be organised in separate folders to encourage reuse and modularity.